src/resolversFactory.js

```diff
@@ -0,0 +1,33 @@
+var util = require('util');
+var http = require('http');
+
+function createWofPipResolver(url) {
+  var baseUrl = url;
+
+  return function(centroid, callback) {
+    var url = util.format('%s/?latitude=%d&longitude=%d', baseUrl, centroid.lat, centroid.lon);
```

Comment:
Looks like too many variables named `url` in different scopes. :smile: 
Could get confusing to follow.